demo/components/my-app.js

```diff
@@ -87,10 +94,12 @@ class MyApp extends connect(store)(HTMLElement) {
     });
 
     installRouter(() => store.dispatch(navigate(window.location)));
-
     installOfflineWatcher((offline) => {
       this._offline.textContent = offline ? ' offline' : 'online';
     });
+    installMediaQueryWatcher(`(min-width: 700px)`, (matches) => {
```

Comment:
change to `600px` to match the comment above, or vice versa.

Reply:
🤦‍♂️